var/spack/repos/builtin/packages/glvis/package.py

```diff
@@ -12,7 +12,7 @@ class Glvis(MakefilePackage):
     homepage = "http://glvis.org"
     git      = "https://github.com/glvis/glvis.git"
 
-    maintainers = ['goxberry', 'v-dobrev', 'tzanio']
+    maintainers = ['goxberry', 'v-dobrev', 'tzanio', 'stitt4']
```

Comment:
Apparently there's no github user with this name...

Reply:
Pretty sure that should be `tomstitt` 😁

Reply:
oops! yes that is not my github name..